cpp/src/matching/subg_match.cu

```diff
@@ -0,0 +1,159 @@
+///#include <gunrock/app/sm/sm_app.cuh>
+#include <thrust/sequence.h>
+
+#include "utilities/graph_utils.cuh"
+#include "utilities/error_utils.h"
+#include <cugraph.h>
+#include <algo_types.h>
+
+#include <iostream>
+#include <type_traits>
+#include <cstdint>
+
+extern double sm(const int            num_nodes,
```

Comment:
I assume this definition is a work-around for their problem of not exposing this call in the current version.

Reply:
Yep. I was hoping I can bypass that and just include the header with the `template<...> sm(...)` but when I try to instantiate it (same way they do in their code) I get compiler errors in their code. You may actually see that `#include <...sm_app.cuh>` is commented out.